### PR TITLE
John

### PR DESCRIPTION
**Pull request by gartee-john-PFG**
I made some magical changes...pls review
@parsontoddPFG @SusmanNickiPFG @hasanimikePFG @andersenjustinPFG @jsfitzhugh @MueggenbergJeffPFG @robinsonwilliam

### CODE REVIEW

File: commitExercise.txt [hunk 1]
```diff
@@ -3,7 +3,11 @@ line 2 main branch
 line 3 main branch
 line 4 main commit 2
 line 5 main commit 2
+line 6 main commit 3
```
Comment (SusmanNickiPFG) [inline commitExercise.txt:6]: @gartee-john-PFG  I'm not sure this is right, maybe try "unicorns!"

File: commitExercise.txt [hunk 2]
```diff
@@ -3,7 +3,11 @@ line 2 main branch
...
+line 7 main commit 3
 line 6 dev commit 1
 line 7 dev commit 1
 line 8 dev commit 2
-line 9 dev. commit 2
\ No newline at end of file
+line 9 dev. commit 2
+line 10 main from nicki
```
Comment (SusmanNickiPFG) [inline commitExercise.txt:12]: Ooooh Fancy!!

File: commitExercise.txt (hunk 1 above)
Comment (gartee-john-PFG) [reply commitExercise.txt:6]: tried it...nope it's a rino

Comment (SusmanNickiPFG) [reply commitExercise.txt:6]: Weird I'd never thought of that before! Have you tried something else with a horn?

**Pull request closed without merging**